source/emit.ts

```diff
@@ -3,6 +3,7 @@ import * as fs from "fs";
 import * as path from "path";
 import prettier from "prettier";
 import { convertSchemaToInterface } from "./convertSchemaToInterface.js";
+import type CustomAttributeConfigurations from "./__snapshots__/responses/query_custom_attribute_configurations.json";
```

Comment:
Didn't know this was possible. But I don't think we should do it, better to type it separately.

Reply:
What is the argument for not doing it?

Remember that when you do `import type` instead of `import`, nothing is actually imported and bundled together (because it's only for the compiler to check the types). So it won't change any output. But it reinforces the strong-typing by a lot.

Reply:
Let me know if (given above) you still want me to change it, and then I will. Just wanted to add my 5 cents first.

Reply:
It makes the CustomAttributeConfiguration type opaque - there's no good way of reading the type. The resolved type in the IDE ends with 
```
| {
    ...;
}
```

Reply:
I see. I never use that actually, I always just care about intellisense and autocompletion. Makes sense though!

I applied the given changes.